builder-hex0.hex2

```diff
@@ -98,8 +99,8 @@
 #                                    { global_file_index, current_offset }
 #    10800 -    1FFFF BIOS memory map
 #    10000 -    107FF interrupt table
-#     A000 -     A1FF sector read buffer - 16bit
-#     7C00 -     8600 code
+#     8E00 -     FFFF unused
+#     7C00 -     8DFF code
```

Comment:
Likewise, this just updates the memory map to account for code size growth in stage2. (When stage1 is not used, code instead ends at 0x8BFF.)